pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "openadapt"
+version = "0.0.0"
+authors = [
+  { name = "Richard Abrich", email = "richard.abrich@mldsai.com" },
+  { name = "Krish Patel", email = "krish@openadapt.ai" },
+  { name = "Mustafa Abdulrahman", email = "mustafa@openadapt.ai" },
+  { name = "Aaron Shah", email = "aaron@openadapt.ai" },
+  { name = "Angela Zhuo", email = "angela@openadapt.ai" },
+  { name = "Avid Eslami", email = "avid@openadapt.ai" },
+  { name = "Dian Rong", email = "dian@openadapt.ai" },
+  { name = "Jesica Susanto", email = "jesica@openadapt.ai" },
+  { name = "Owais Zahid", email = "owais@openadapt.ai" },
+  { name = "Andrew Gorton", email = "andrew.gorton@mail.utoronto.ca" }
```

Comment:
@KrishPatel13 what do you think about listing these alphabetically by name?
Also the format here does not match https://python-poetry.org/docs/pyproject/#authors , can you please clarify?

Reply:
I followed this [convention](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#authors-maintainers), I can also reorder these alphabetically
![image](https://github.com/MLDSAI/OpenAdapt/assets/43456930/c0ee5fb1-f6dd-4dda-b21b-f3a9cdac1120)

Reply:
Interesting, thank you for clarifying! Not sure what the discrepancy is all about, let's stick with the official docs as you've done here 👍 

Alphabetically by last name sounds good 👍